src/dependency-injection/buildBaseContainer.js

```diff
@@ -1,5 +1,16 @@
 import MiddlewareRegistry, { GLOBAL_CONTEXT } from './MiddlewareRegistry';
-import { compose } from 'redux';
+
+function compose(...funcs) {
```

Comment:
Do you want to add a comment saying you've borrowed this from redux, just for clarity?